tools/bats/test_start.bats

```diff
@@ -0,0 +1,110 @@
+#!/usr/bin/env bats
+
+# Load the functions from start.sh
+source './start.sh'
```

Comment:
this is ../../ two steps up

Reply:
The tests run in the same directory as start.sh.